recipes/sos-python/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "sos-python" %}
+{% set version = "0.18.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 360dd16db42664d3a609037354bd92c435a246510079621c906be9203338051c
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - sos-notebook >=0.19.4
```

```suggestion
    - sos-notebook >=0.19.4
    - sos >=0.19.8
```
https://github.com/vatlab/sos-python/blob/a2887785f50373fb234ad56d77f8c32c41c2e083/setup.py#L42